Block following Case renders double newlines

```go
func ExampleCaseBug() {
	c := Switch(Id("a")).Block(
		Case(Lit(1)).Block(
			Var().Id("i").Int(),
			Var().Id("j").Int(),
		),
	)
	fmt.Printf("%#v", c)
	// Output:
	// switch a {
	// case 1:
	// 	var i int
	// 	var j int
	// }
}
```

Result:
```
=== RUN   ExampleCaseBug
--- FAIL: ExampleCaseBug (0.00s)
got:
switch a {
case 1:
	var i int

	var j int
}
want:
switch a {
case 1:
	var i int
	var j int
}
FAIL
```

--- a/jen/examples_test.go
+++ b/jen/examples_test.go
@@ -8,6 +8,22 @@ import (
 	. "github.com/dave/jennifer/jen"
 )
 
+func ExampleCaseBug() {
+	c := Switch(Id("a")).Block(
+		Case(Lit(1)).Block(
+			Var().Id("i").Int(),
+			Var().Id("j").Int(),
+		),
+	)
+	fmt.Printf("%#v", c)
+	// Output:
+	// switch a {
+	// case 1:
+	// 	var i int
+	// 	var j int
+	// }
+}
+
 func ExampleCustom() {
 	multiLineCall := Options{
 		Close:     ")",

--- a/jen/group.go
+++ b/jen/group.go
@@ -43,7 +43,6 @@ func (g *Group) render(f *File, w io.Writer, s *Statement) error {
 		if isGrp && grp.name == "case" || isTkn && tkn.content == "default" {
 			g.open = ""
 			g.close = ""
-			g.separator = "\n"
 		}
 	}
 	if g.open != "" {

